internal/output/interleaved.go

```diff
@@ -6,6 +6,6 @@ import (
 
 type Interleaved struct{}
```

Comment:
Note, the `Interleaved`/`Prefixed`/`Group` types are no longer used outside of the `output` package (they're now only created via `output.Style.Build()` and only consumed as the `output.Output` interface), so they could be unexported